test/jdl/validators/relationship-validator.spec.js

```diff
@@ -168,27 +168,6 @@ describe('RelationshipValidator', () => {
                     });
                 });
             });
-            context(`when having a ${MANY_TO_ONE} relationship`, () => {
-                context('when having a bidirectional relationship', () => {
-                    let relationship;
-
-                    before(() => {
-                        relationship = new JDLRelationship({
-                            from: 'A',
-                            to: 'B',
-                            type: MANY_TO_ONE,
-                            injectedFieldInFrom: 'b',
-                            injectedFieldInTo: 'a',
-                        });
-                    });
-
-                    it('should fail', () => {
```

Comment:
You should invert the test instead of removing.
`should fail` to `should add the relationship`

Reply:
fixed